doc/topics/cloud/index.rst

```diff
@@ -7,120 +7,54 @@ Salt Cloud
-      # Optional: Set up the location of the Salt Master
-      #
-      minion:
-        master: saltmaster.example.com
-
-The required configuration varies between cloud hosts so make sure you read the provider specifics.
-
-List Cloud Provider Options
----------------------------
-
-You can now query the cloud provider you configured for available locations,
-images, and sizes. This information is used when you set up VM profiles.
-
-.. code-block:: bash
-
-    salt-cloud --list-locations <provider_name>  # my-ec2 in the previous example
-    salt-cloud --list-images <provider_name>
-    salt-cloud --list-sizes <provider_name>
-
-Replace ``<provider_name>`` with the name of the provider configuration you defined.
-
-Create VM Profiles
-------------------
-
-On your Salt Master, browse to ``/etc/salt/cloud.profiles.d/`` and create a file called ``<provider>.profiles.conf``,
-replacing ``<provider>`` with ``ec2``, ``softlayer``, and so on. The file must end in ``.conf``.
-
-You can now add any custom profiles you'd like to define to this file. Here are a few examples:
-
-.. code-block:: yaml
-
-    micro_ec2:
-      provider: my-ec2
-      image: ami-d514f291
-      size: t1.micro
-
-    medium_ec2:
-      provider: my-ec2
-      image: ami-d514f291
-      size: m3.medium
-
-    large_ec2:
-      provider: my-ec2
-      image: ami-d514f291
-      size: m3.large
-
-Notice that the ``provider`` in our profile matches the provider name that we defined? That is how Salt Cloud
-knows how to connect to create a VM with these attributes.
-
-Create VMs
-----------
-
-VMs are created by calling ``salt-cloud`` with the following options:
-
-.. code-block:: bash
-
-    salt-cloud -p <profile> <name1> <name2> ...
-
-For example:
-
-.. code-block:: bash
-
-    salt-cloud -p micro_ec2 minion1 minion2
-
-Destroy VMs
------------
-
-Add a ``-d`` and the minion name you provided to destroy:
-
-.. code-block:: bash
-
-    salt-cloud -d minion1 minion2
-
-Query VMs
----------
-
-You can view details about the VMs you've created using ``--query``:
-
-.. code-block:: bash
-
-    salt-cloud --query
+Configuration
+=============
+Salt Cloud provides a powerful interface to interact with cloud hosts. This
+interface is tightly integrated with Salt, and new virtual machines
+are automatically connected to your Salt master after creation.
+
+Since Salt Cloud is designed to be an automated system, most configuration
+is done using the following YAML configuration files:
+
+- ``/etc/salt/cloud``: The main configuration file, contains global settings
+  that apply to all cloud hosts. See :ref:`Salt Cloud Configuration
+  <salt-cloud-config>`.
+
+- ``/etc/salt/cloud.providers.d/*.provider.conf``: Contains settings that
+  configure a specific cloud host, such as credentials, region settings, and so
+  on. Since configuration varies significantly between each cloud host,
+  a separate file is created for each cloud host. In Salt Cloud, a provider is
```

Comment:
How about "a separate file needs to be created" - the way this is written now seems to imply that salt or salt-cloud creates the config files automatically, which it does not.